README.md

```diff
@@ -85,6 +85,10 @@ In the `./app` folder, create a `.env` file with the following variable:
   `VITE_BANK_IMPL_ADDRESS=0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9`
 - **VITE_APP_VITE_BANK_BEACON_ADDRESS**=: The bank beacon address
   `VITE_BANK_IMPL_ADDRESS=0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9`
+- **VITE_APP_VOTING_IMPL_ADDRESS**=: The voting implementation address
+  `VITE_APP_VOTING_IMPL_ADDRESS=0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9`
+- **VITE_APP_VOTING_BEACON_ADDRESS**=: The voting beacon address
+  `VITE_APP_VOTING_BEACON_ADDRESS=0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9`
 - _VITE_APP_NETWORK_ALIAS_: The string identifier of an EVM compatible network that the app uses. Example: `VITE_APP_NETWORK_ALIAS=polygon`. This variable is optional but if you don't set your own network parameters it has to be provided. Use this if you want to use one of the preset networks which the application provides. Available options are:
```

Comment:
The value here are different from the value in the ENV_LIST

Reply:
which one is different?